src/Sync.js

```diff
@@ -70,8 +70,10 @@ class Sync {
   }
 
   async getRepos() {
+    let repositories = [];
     await Promise.all(
```

Comment:
You could just return the repository at the end of the map function and then 

`return await Promise.all(...`

Reply:
you dont have to `return await` - just returning the promise is enough